ovn_k8s/modes/overlay.py

```diff
@@ -95,11 +92,9 @@ def _create_load_balancer_vip(self, service_type, service_ip, ips, port,
             return
 
         # target is of the form "IP1:port, IP2:port, IP3:port"
-        target = ""
-        for ip in ips:
-            target = target + ip + ":" + str(target_port) + ","
-        target = target[:-1]
-        target = "\"" + target + "\""
+        target_endpoints = ",".join(["%s:%s" % (ip, target_port)
+                                     for ip in ips])
```

Comment:
For some reason, I have never been able to internalize the above Python possibility.